Heading/Heading.module.less

```diff
@@ -0,0 +1,86 @@
+// Heading.module.less
+//
+@import "../styles/variables.less";
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.heading {
+	.applySkins({
+		border-bottom: @agate-heading-border-width solid transparent;
+		margin: 0 @agate-spotlight-outset (1.5 * @agate-spotlight-outset) @agate-spotlight-outset;
+		padding: @agate-heading-padding;
+		min-height: (1.5 * @agate-spotlight-outset);
+		letter-spacing: @agate-heading-letter-spacing;
+		line-height: @agate-heading-line-height;
+		color: @agate-heading-text-color;
+
+		&.showLine {
+			border-bottom-color: @agate-heading-border-color;
+			padding: @agate-heading-showline-padding;
+		}
+
+		&.title {
+			font-family: @agate-heading-title-font-family;
+			font-weight: @agate-heading-title-font-weight;
+			font-style: @agate-heading-title-font-style;
+			font-size: @agate-heading-title-font-size;
+			position: relative;
+			padding: @agate-heading-title-padding;
+
+			&::after {
+				content: "";
+				position: absolute;
+				height: @agate-heading-title-decoration-height;
+				width: @agate-heading-title-decoration-width;
+				.position(@agate-heading-title-decoration-positions);
+				background-color: var(--agate-heading-accent, @agate-heading-accent-color);
```

Comment:
`@agate-heading-accent-color` isn't defined in the colors file

Reply:
Crap... it's defined in the other ticket. :ughwhy:. I'll import it.